syntax="proto3";

package TrZeroMQMsg;

option go_package="./TrZeroMQMsgpb";

message TopData{
    //检索的类型
    string type=1;
    //序列化后的字符串
    bytes rawData=2;
}